### PR TITLE
Update pt.lua

### PR DESCRIPTION
**Pull request by catiagomes94a**
**Describe Pull request**
Missing phrases on translate of portuguese (pt) so i decided fix this.

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes]
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]

### CODE REVIEW

File: locales/pt.lua [hunk 1]
```diff
@@ -36,39 +38,45 @@ local Translations = {
...
         ["stash"] = "Baú",
-        ["change_outfit"] = "Alterar Outfit",
+        ["change_outfit"] = "Mudar outfit",
         ["outfits"] = "Outfits",
-        ["change_character"] = "Alterar Personagem",
+        ["change_character"] = "Alterar personagem",
         ["characters"] = "Personagens",
-        ["enter_unlocked_house"] = "Entrar Na Casa",
-        ["lock_door_police"] = "Trancar Porta"
+        ["enter_unlocked_house"] = "Entrar na casa,
```
Comment (D4isDAVID) [inline locales/pt.lua:67]:
```suggestion
        ["enter_unlocked_house"] = "Entrar na casa",
```
Finish the string here

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

File: locales/pt.lua (hunk 1 above)
Comment (catiagomes94a) [reply locales/pt.lua]: ty

**Review by D4isDAVID — APPROVED** [review]
LGTM

**Comment by rodrigoandrade1** [comment]
Fix realestate_only translation please

**Pull request merged**